### DISCUSSION TITLE
Watchlist and integration into Jellyfin

### DISCUSSION THREAD

**Discussion by samiraa78** [seerr-team/seerr#437] [Ideas]
In Sonarr we have a show to add to watchlist where in new episode or season will download as and when they are available can we have the same option in Jellyseerr

Also it would be great if we can have some sort of integration into Jellyfin where we can browse shows or movies in Jellyfin and request from there itself why this is important because most users will use Jellyfin on a tv so it would be difficult for them to go through jellyseerr

**Comment by fallenbagel** [comment]
We already have watchlists. And it already does that even without watchlists? As you can already have media requested thats not available so its monitored on sonarr when available.

Browsing through jellyfin is impossible without changing jellyfin code because modifying the website through plugins is impossible

**Reply by samiraa78** [reply]
Thanks for your reply

May be I am missing out something
When I search for a series I can only request it by selecting existing season and downloading it but if I was to select the upcoming season so that it is added to sonar and then downloaded as an when it is available I cannot do that

Something I can do in sonar I would want to do that is for many reasons being old seasons have been watched and hence to save space and bandwidth I don't want them to be downloaded

I understand I can go to sonar and do it but then I have to keep using two apps

**Reply by fallenbagel** [reply]
You mean only request future episodds? Not request current episodes and have future monitored?

**Reply by samiraa78** [reply]
Yes

**Reply by DinleyH** [reply]
Dam wish I checked this before i spent so much time trying to make a jellyfin plugin that allows media requests ! ah well. I have been trying to find ways to integrate to jellyfin for ages now but to no avail! 

Closest I can get is adding a link to jellyseer on jellyfins sidebar hahah. Or an iframe? doubt that would work tbh.
